CC-21940 Propel migration rollback. (#1497)
spryker/spryker-docs: docs/scos/dev/back-end-development/zed/persistence-layer/database-schema-definition.md

This migration is applied only once and is registered in the `propel_migration table`.

## Migrate to specific version

{% info_block warningBox "Warning" %}

> Functionality is available only with [Spryker Core](/docs/scos/dev/feature-integration-guides/202304.0/spryker-core-feature-integration.html) `202304.0` or higher.

Author: @dmytro-sokolovskyi, discussed possible solutions with @vad1msa4. It was decided to remove the link instead of copying IG without changes.

{% endinfo_block %}

Spryker allows migration to a specific version by using the `migrate-to-version` option with the `propel:migrate` command. The option defines the version to migrate the database to and expects value that can be taken from the `version` column in the `propel_migration` table.

For example:
```bash
vendor/bin/console propel:migrate --migrate-to-version=1622797441
```

> Depending on the current database version, the command will either migrate or rollback to the given version.

Reviewer: ```suggestion
Depending on the current database version, the command either migrates or rolls back to the given version.
```

> In order to check the current version, the `propel:migration:check` command with the `last-version` option can be used. The option allows receiving the version of the last executed migration.

Reviewer: ```suggestion
To check the current version, use the `propel:migration:check` command with the `last-version` option. The option lets you receive the version of the last executed migration.
```
